projections/weather/lib/weather/client.rb

```diff
@@ -15,25 +15,40 @@ def initialize(*args)
       super
     end
 
-    # TODO!
+    # @example Request syntax with placeholder values
+    # 
+    #   output = client.get_city(
+    #     city_id: "CityId", # required
+    #   )
```

Comment:
Do these need an empty comment line after for rdoc to render correclty?

Reply:
I'm not sure but that can be added later. I need to do a response example
 I am thinking of doing a to_h representation instead of method calls. I could reuse this code or do both examples in this class. Let me know what you think.